test/TPP/passes/mlp-packed.mlir

```diff
@@ -0,0 +1,65 @@
+// RUN: tpp-opt %s -pack-matmul="block-factors=32,32,32" -map-linalg-to-tpp -transform-dialect-interpreter -canonicalize -tile-consumer-and-fuse-producers="tile-sizes=1,0,0,0" -canonicalize -tile-consumer-and-fuse-producers="tile-sizes=1,0,0" -canonicalize -empty-tensor-to-alloc-tensor -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" -canonicalize -drop-equivalent-buffer-results -finalizing-bufferize -canonicalize -map-linalg-to-tpp -convert-linalg-to-tpp="use-parallel-loops=false" -map-to-brgemm -convert-linalg-to-tpp | FileCheck %s
```

Comment:
Duplicated use of `tile-consumer-and-fuse-producers` in this RUN line

Reply:
It is not duplicated. We have two rounds of fusion

Reply:
I see. Weird, but fine, we can worry about that later if needed.